### DISCUSSION TITLE
LXDE project - it is dead? If not what person from LXQt are active in this project? I have some issues to report but developers didn't answer me. I have also some suggestion.

### DISCUSSION THREAD

**Discussion by motocyklista** [lxqt/lxqt#1989] [General]
Dear LXQt Developers,

Like we know some people from LXQt are from LXDE project. I can see that LXDE website is still active and some distributions like Fedora 34 still have LXDE version. I have questions and some suggestion but I can't contact with LXDE developers. Maybe someone from LXQt is still active in LXDE project? Maybe Discussions section for github website in LXDE should be also available like in LXQt?

I have the same issue with mouse settings (sensitivity, acceleration) in LXDE like was affected in lxqt-panel.

LXQt is more resource hungry than LXDE so this is why I still stay with LXDE. I have installed LXQt on my machine but this was slower. 

I have also question to LXDE. This still use GTK2? I found on some websites that GTK2 is dead now because GTK4 is available and GTK2 was dropped. Is this true? If yes this means that LXDE is not safe anymore? Maybe some people from LXDE can answer me?

I will be apreciate for answer and help.

**Comment by tsujan** [comment]
LXQt has no relation to LXDE. They're totally separate projects. We don't know about LXDE.

You need to report at https://github.com/lxde; whether they'll respond or not is out of our control but a look at that site suggests that, although LXDE isn't dead, it isn't lively either.

**Reply by motocyklista** [reply]
@tsujan I know that LXQt it is not LXDE and both are separated projects but why I asked in this place in LXQt? Because I still can see a lot of people mentioned in LXDE github website. If you look at "People" section here I can see that you are also mentioned --> https://github.com/orgs/lxde/people

So if you @tsujan are not working in LXDE project why you are mentioned? Can you explain me?

I hope that you will answer me.


P.S. Do you know who is a leader of the project LXDE? Is this @LStranger?

**Reply by tsujan** [reply]
> So if you @tsujan are not working in LXDE project why you are mentioned?

→ https://github.com/lxqt/lxqt-config/issues/400#issuecomment-832371603

LXQt was started when LXDE devs became dissatisfied with the direction GTK3 went in. So, it's understandable that you may find the names of most LXQt members in LXDE. Some devs joined later (I was a KDE user).

> Do you know who is a leader of the project LXDE? Is this @LStranger?

I thinks so.

**Reply by motocyklista** [reply]
@tsujan I am new on github and my english is not the best so you must forgive me. If you are not active member in LXDE so it is option to leave LXDE project website on github and make that you are will not be mentioned in "People" section? I asked you about it because it is very interesting for me. Thanks for your opinion and answer.

**Reply by tsujan** [reply]
I have no admin permission there. I just made my membership private minutes ago.

**Reply by motocyklista** [reply]
@tsujan thank you for your answer and explain. This is very helpful for me. I have also question to LXQt. Like we know LXQt is active developed and we have sometimes new version release. I would like to now if new version have always higher hardware requirements? I tried LXQt from Lubuntu 20.04.2 and this was LXQt 0.14.1. I noticed that is slower than LXDE included in Lubuntu 18.04. Maybe new version of LXQt is faster than 0.14.1? I want to have only similar performance to LXDE. I have very old machine. Did you tested new version of LXQt for performance?

**Reply by tsujan** [reply]
As far as I've seen, all LXQt members care for RAM and CPU. The codes are being optimized continuously. Whether LXQt works on a very old machine depends on whether Linux does.

Please also note that we aren't responsible for Lubuntu or any other distro.

**Reply by motocyklista** [reply]
@tsujan I know that you are not responsible for Lubuntu or other distro but I thinked that you can check performance for each LXQt version. Is very interesting for me because maybe new version are more resource hungry. This is very common on other graphic environments like GNOME or XFCE (xfce is more resource hungry because this use GTK3).

LXDE now works perfectly fine but I asked on their website and I can see that no reaction so probably this project is not active so I am very interseting about LXQt. I have question to GTK3 programs like GNOME DISK UTILITY. This will be work on LXQt? Like we know LXQt use plasma so programs from GTK3 will be work fine without issue?

**Reply by stefonarch** [reply]
LXQt doesn't _use plasma_  but qt libraries for itself and you can install and use any application you like, depending only on the distro you use.

**Reply by tsujan** [reply]
> This is very common on other graphic environments...

That's completely normal and LXQt isn't an exception. When LXQt devs say that LXQt will remain light, they don't mean that its resource usage won't increase. A light DE in 2021 is heavier than a light DE in 2014.

> ...GNOME DISK UTILITY. This will be work on LXQt?

yes.

> Like we know LXQt use plasma

It doesn't. LXQt is based on Qt. "Plasma" is a KDE term, used for KDE's desktop.

**Comment by motocyklista** [comment]
I have other question. I have now installed LXDE and LXAppearance 0.6.3. I very like default white cursor from LXDE. Can I import it from this to LXQt? Where are located necessary files in LXDE and LXQt. I want to copy these files to LXQt but I must know if it is compatible to install cursor from LXDE to LXQt? Thank you for help.

P.S. I think that I found this for Ubuntu, here localization of the files --> /usr/share/icons but I don't know if it will be work with no issues.

**Reply by tsujan** [reply]
Any cursor theme can be used in LXQt.

> I want to copy these files to LXQt

As I said, LXQt has no relation to LXDE. As you can't copy GNOME settings to KDE, you can't either with LXDE and LXQt.

**Comment by motocyklista** [comment]
I have some question to LXQt. Do you have polish language support for LXQt? You have people who help with translate to polish? If not maybe I can help but my english is not the best. Of course it is not disaster but not perfect.

**Reply by stefonarch** [reply]
Translations can be done in our weblate instance, open an account and pick the untranslated elements:
https://translate.lxqt-project.org/languages/pl/

**Reply by motocyklista** [reply]
@stefonarch thank you for information. Is it only option to do with above website? I can't do this here on github?

I have also another question to LXQt. Do you know where I will find configuration files for autostart (autostart for programs)? I know that it is option to do this by GUI lxqt-config-session but I don't know where I can find files to edit this "manually".

**Reply by stefonarch** [reply]
> @stefonarch thank you for information. Is it only option to do with above website? I can't do this here on github?

Github PR's are problematic, could lead to merge conflicts and there is no check for mistakes, therefor using weblate is quite mandatory now.


> I don't know where I can find files to edit this "manually"

`~/.config/autostart`
